config/services.yaml

```diff
@@ -21,7 +26,7 @@ services:
             $emailSender: '%app.notifications.email_sender%'
             $projectDir: '%kernel.project_dir%'
             $publicFolder: '%bolt.public_folder%'
-            $tablePrefix: '%bolt.table_prefix%'
+            $tablePrefixes: '%app.table_prefix%'
```

Comment:
If we could leave this as `$tablePrefix: '%app.table_prefix%'`, then we could probably make this change so it won't break for people who are updating.